### ISSUE TITLE
logging in with a deleted account doesn't work

### ISSUE THREAD

**Issue by 32th-System** [n-rook/thscoreboard#283]

**Comment by 32th-System** [comment]
Should logging in with a deleted account show some kind of "You have logged in with an account you chose to delete. Do you wish to restore your account" yes no prompt or should logging in with a deleted account simply restore the account?

**Comment by n-rook** [comment]
Definitely the former; we wouldn't want people being surprised their accounts are still around after they committed to deleting them.

This would be good to have but it's not really a bug I think

**Comment by 32th-System** [comment]
https://github.com/n-rook/thscoreboard/issues/203#issuecomment-1419687498
> [...] While the account is "to be deleted", the user can still log in, but the only thing they should be able to do is tell us they changed their mind and don't want the account deleted anymore.

**Comment by n-rook** [comment]
yeah but that's just a missing feature; it's not like the public website implies that's implemented right now
